File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/tx-modifiers.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
	{
        Fscc.Port p = new Fscc.Port(0);

        var modifiers = p.TxModifiers;

        // Enable transmit repeat & transmit on timer
        p.TxModifiers = Fscc.TxModifiers.TXT | Fscc.TxModifiers.XREP;

        // Revert to normal transmission
        p.TxModifiers = Fscc.TxModifiers.XF;

        return 0;
    }
}

File: Drivers/FSCC Drivers/windows/lib/fscc/c++/examples/tx-modifiers.cpp
#include <fscc.hpp> // Fscc::Port

int main(void)
{
    Fscc::Port p(0);

    unsigned modifiers = p.GetTxModifiers();

    // Enable transmit repeat & transmit on timer
    p.SetTxModifiers(Fscc::TxModifiers::TXT | Fscc::TxModifiers::XREP);

    // Revert to normal transmission 
    p.SetTxModifiers(Fscc::TxModifiers::XF);

    return 0;
}